plugin-template

```diff
@@ -65,6 +65,8 @@ DEPRECATED_FILES = [
     ".github/workflows/fips.yml",
     ".travis",
     ".travis.yml",
+    "docs/_static/.this_makes_it_so_git_keeps_the__static_folder",
+    "docs/_static/api.json"
```

Comment:
```suggestion
    # Those files are deprecated from the docs template section. It would be unsafe to delete them without reapplying the latter.
    # "docs/_static/.this_makes_it_so_git_keeps_the__static_folder",
    # "docs/_static/api.json"
```

I agree that it's unsafe today. But it would be a shame to forget about them.

Reply:
Now seperated by section.